rancher/charts
# [dev-v2.8] Bump rancher-monitoring/rancher-monitoring to use new rancher/shell

## Issue: 
SURE-7080

## Problem

See original issue


## Solution

See original issue

## Testing


## Engineering Testing
### Manual Testing

Installed `103.0.3+up45.31.1` via Apps, upgraded to `103.0.4+up45.31.1`, then subsequently to `103.0.5-rc1+up45.31.1`.
Upon upgrade to `103.0.5-rc1+up45.31.1` I can observe shell `v0.1.23` in use as expected.

### Automated Testing


## QA Testing Considerations


### Regressions Considerations


## Backporting considerations
